schemas/multimodal/build.py

```
@@ -0,0 +1,237 @@
+                f"--python_out={python_out_dir}",
+                f"--es_out={TS_PLUGIN_OPTION}:{ts_out_dir}",
+                str(proto_path),
+            ]
+        )
+
+        python_generated = python_out_dir / PYTHON_GENERATED_NAME
+        ts_generated = ts_out_dir / TS_GENERATED_NAME
+        if not python_generated.exists():
+            raise FileNotFoundError(
+                f"Missing generated Python protobuf module: {python_generated}"
+            )
```

Ruff (0.15.11)

[warning] Avoid specifying long messages outside the exception class

(TRY003)